prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
Oh sorry I misread. Yeah go ahead. I can also do that if you don't know where to look at.